### PR TITLE
Sync serverless ci

### PR DESCRIPTION
**Pull request by aliok**
Sync serverless ci - based on https://github.com/openshift-knative/hack main branch

cc @pierDipi @creydr

### CODE REVIEW

**Comment by pierDipi** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-kafka-broker-release-v1.8-410-images

**Review by creydr — APPROVED** [review]
/lgtm

File: ci-operator/config/openshift-knative/eventing/openshift-knative-eventing-release-next__410.yaml [hunk 1]
```diff
@@ -1,186 +1,6 @@
-base_images:
```
Comment (pierDipi) [inline ci-operator/config/openshift-knative/eventing/openshift-knative-eventing-release-next__410.yaml:1]: This deletion doesn't seem correct, it's probably related to my recent changes on release-next. I will take a look

Comment (pierDipi) [reply ci-operator/config/openshift-knative/eventing/openshift-knative-eventing-release-next__410.yaml:1]: https://github.com/openshift-knative/eventing/pull/166

Comment (pierDipi) [reply ci-operator/config/openshift-knative/eventing/openshift-knative-eventing-release-next__410.yaml:1]: Can you rerun it?

Comment (aliok) [reply ci-operator/config/openshift-knative/eventing/openshift-knative-eventing-release-next__410.yaml:1]: I am now running

**Comment by aliok** [comment]
@pierDipi I re ran now

**Comment by pierDipi** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-kafka-broker-release-v1.8-412-images

**Comment by aliok** [comment]
/pj-rehearse ack

oh man

**Comment by aliok** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-next-412-images
/pj-rehearse pull-ci-openshift-knative-eventing-release-next-410-images

**Comment by pierDipi** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-next-412-images

**Pull request merged**